bots/hermes/config.py

```diff
@@ -0,0 +1,7 @@
+from configparser import ConfigParser
+
+config = ConfigParser()
+config.read('./.secrets/secrets.ini')
```

Comment:
Can you add in the repository a `secrets.example.ini` file that contains a example of how the file should be structured ?

Like we did in this file 
https://github.com/osscameroon/osscameroon-website/blob/main/scraper/github/example.config.txt

Reply:
Can you put this `secrets.ini` file outside of the `.secrets` folder and name it `config.ini` instead ?

Also you should consider the newly created `config.ini` file in the `.gitignore` file so that he does not get pushed to git :)

Reply:
Now that you have renamed the file from `secrets.ini` to `config.ini` you should probably change this that file name here as well